boards/holybro/durandal-v1/default.cmake

```diff
@@ -0,0 +1,126 @@
+
+px4_add_board(
+	PLATFORM nuttx
+	VENDOR holybro
+	MODEL durandal-v1
```

Comment:
Is `v1` actually in the name somewhere or is this our addition?

Reply:
F.Y.I. The next Durandal would be named Durandal 2.

Reply:
@dagar I added it in anticipation